app/Http/Controllers/Front/SolutionController.php

```diff
@@ -124,14 +130,48 @@ public function submit($slug, Request $request)
 
                     $attributes['data'][$sectionIndex][$fieldIndex]['value'] = $fileName;
                 }
+
+                /** Search for the name */
+                if ((false !== strpos(strtolower($field['label']), 'nume')) &&
+                    (false !== strpos(strtolower($field['label']), 'prenume'))) {
+                    $name = $field['value'];
+                }
+
+                /** Search for the email */
+                if ((false !== strpos(strtolower($field['label']), 'e-mail')) ||
+                    (false !== strpos(strtolower($field['label']), 'email'))) {
+                    $email = $field['value'];
+                }
+            }
+        }
+
+        /** Extract user or create a new one. */
+        $user = null;
+        if ($email) {
+            $user = DashboardUser::whereEmail($email)->first();
+
+            if (null === $user) {
+                $user = new DashboardUser();
+
+                $user->fill([
+                    'email'     => $email,
+                    'name'      => $name,
+                    'user_role' => 'applicant',
+                    'published' => 1,
+                ]);
```

Comment:
Can be simplified to:
```suggestion
                $user = new DashboardUser([
                    'email'     => $email,
                    'name'      => $name,
                    'user_role' => 'applicant',
                    'published' => 1,
                ]);
```